scripts/dev-setup.sh

```
@@ -0,0 +1,60 @@
+
+# Check if environment file exists
+if [ ! -f ".env.local" ]; then
+    echo "⚠️  No .env.local file found. Creating from template..."
+    cp env.example .env.local
+    echo "📝 Please edit .env.local with your actual configuration values"
+fi
+
+# Start Firebase emulators in background
+echo "🔥 Starting Firebase emulators..."
+firebase emulators:start --only auth,firestore --project demo-project &
+FIREBASE_PID=$!
```

Shellcheck (0.10.0)

[warning] FIREBASE_PID appears unused. Verify use (or export if used externally).

(SC2034)